files/build_templates/sonic_debian_extension.j2

```diff
@@ -145,18 +145,15 @@ sudo cp -f $IMAGE_CONFIGS/cron.d/* $FILESYSTEM_ROOT/etc/cron.d/
 
 # Copy NTP configuration files and templates
 sudo cp $IMAGE_CONFIGS/ntp/ntp-config.service $FILESYSTEM_ROOT/etc/systemd/system/
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable ntp-config.service
```

Comment:
keep

Reply:
Moving to generator because of dependency on updategraph